pkg/sequencediagram/sequencediagram_test.go

```diff
@@ -61,6 +61,45 @@ deactivate _0
 	assert.Equal(t, expected, r)
 }
 
+func TestSequenceDiagramHidden(t *testing.T) {
+	t.Parallel()
+
+	logger, _ := test.NewNullLogger()
+	m, err := parse.NewParser().Parse("tests/hidden.sysl",
+		syslutil.NewChrootFs(afero.NewOsFs(), projDir))
+	require.NoError(t, err)
+	l := &cmdutils.Labeler{}
+	p := &SequenceDiagParam{Title: "Profile"}
+	p.Endpoints = []string{"App <- Endpoint1"}
+	p.AppLabeler = l
+	p.EndpointLabeler = l
+	r, err := GenerateSequenceDiag(m, p, logger)
+	assert.NotNil(t, m)
+	assert.NotNil(t, r)
+	assert.Nil(t, err)
+	expected := `''''''''''''''''''''''''''''''''''''''''''
```

Comment:
Thoughts on doing a `assert.True(t, strings.Contains(r, "Endpoint3"))` instead? That means you can easily change the input sysl file in future tests

Reply:
I'd rather have a rigid testcase that requires an exact match, otherwise there's the risk that the testcase "fails" but doesn't actually fail the test